js/views/messagesview.js

```diff
@@ -194,6 +195,15 @@ define(function(require) {
 				return;
 			}
 		},
+		onWheel: function(event) {
+			if (event.originalEvent.wheelDelta && event.originalEvent.wheelDelta > 0) {
+				// Scrolling up in non-FF browsers
+				this.onScroll();
+			} else if (event.originalEvent.detail && event.originalEvent.detail < 0) {
+				// Scrolling up in FF
+				this.onScroll();
```

Comment:
Is this supposed to be the same lines (201 & 204)?
Because I feel like whatever if we got, the result is always `this.onScroll();` So I don't see why this is needed! :)